src/charts/donut/examples/donut-pct-chart.js

```diff
@@ -26,6 +26,15 @@
  * <li>.tooltipFn(d)   - user defined function to customize the tool tip (optional)
  * <li>.centerLabelFn  - user defined function to customize the text of the center label (optional)
  * <li>.onClickFn(d,i) - user defined function to handle when donut arc is clicked upon.
+ * <li>.label          - object containing properties for external label (optional)
```

Comment:
Hi, I don't think ```.label``` should be under config, rather it should be a separate param like ```center-label``` is, and called ```side-label``` with properties orientation, title, and perhaps something like 'amount-type' or 'units' for available, percent, used.